app/src/main/res/layout/fragment_exposure_notification_list.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="android.view.View" />
+        <variable name="model" type="fi.thl.koronahaavi.exposure.ExposureDetailViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.core.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:padding="20dp"
+            android:clipToPadding="false">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <TextView
```

Comment:
A3+max screen zoom+largest font: when exposures bottom sheet is opened the title ("Exposure notifications") and text "Last checked today ..." aren't visible, but instead first exposure item's top edge is aligned with the bottom sheet (that isn't fully expanded) top edge. The "last checked..." text is somewhat useful info so it would be nicer if initially the top of of the sheet content would be fully visible.